docs/kb/semgrep-appsec-platform/saml-google-workspace.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Learn how to set up SAML access to Semgrep AppSec Platform with Google Workspace.
+tags:
+  - Semgrep AppSec Platform
+  - SSO
+---
+
+# SAML Single Sign-on with Google Workspace
+
+The process of setting up SAML Single Sign-on for the Semgrep AppSec Platform with Google Workspace is fundamentally similar to the process for other providers. However, there are a few quirks of Google Workspace to account for.
+
+Follow these steps:
+
+1. [Set up a custom SAML app](https://support.google.com/a/answer/6087519?hl=en#zippy=%2Cstep-add-the-custom-saml-app) in Google Workspace. The default Name ID, primary email, is optimal for Semgrep AppSec Platform.
+2. When you reach step 12, **Add mapping**, add the two attribute statements that Semgrep AppSec Platform requires: `name` and `email`.
```

Comment:
```suggestion
2. When you reach the **Add mapping** step of the instructions to [Set up a custom SAML app](https://support.google.com/a/answer/6087519?hl=en#zippy=%2Cstep-add-the-custom-saml-app), add the two attribute statements that Semgrep AppSec Platform requires: `name` and `email`.
```

Rewriting this a bit in case the step number ever changes

Reply:
I like that part, but the extra link seems weird to me since we just linked in the previous line. I would assume the person has the Google steps open here already. What do you think about that?

Reply:
That's fair; I updated the suggestion to omit the link